src/plans/stepsize.jl

```diff
@@ -756,6 +757,64 @@ function show(io::IO, a::NonmonotoneLinesearch)
 end
 get_message(a::NonmonotoneLinesearch) = a.message
 
+@doc raw"""
+    PolyakStepsize <: Linesearch
+
+
+Compute a step size
+
+````math
+α_i = \frac{f(p^{(i)}) - f_{\text{best}}+γ_k}{\lVert{ ∂f(p^{(i)})} \rVert^2},
+````
+
+where ``f_{\text{best}}`` is the best cost value seen until the current iteration,
+and ``γ_i`` is a sequence of numbers that is square summable, but not summable (the sum must diverge to infinity).
+Furthermore ``∂f`` denotes a subgradient of ``f`` at the current iterate ``p^{(k)}``.
```

Comment:
```suggestion
Furthermore ``∂f`` denotes a subgradient of ``f`` at the current iterate ``p^{(i)}``.
```

Reply:
Oh, one might notice, I am just on my first weekend after a stressful semester.

Reply:
Don't worry, maybe a little bit of rest would be a good idea then :slightly_smiling_face:

Reply:
Went on a longer bike ride this morning already, so I am on my way there :)